pkg/sanity/volume.go

```diff
@@ -775,9 +758,8 @@ var _ = Describe("Volume [Volume Tests]", func() {
 			err = volumedriver.Delete(volumeID)
 			Expect(err).ToNot(HaveOccurred())
 
-			volumes, err := volumedriver.Enumerate(&api.VolumeLocator{}, nil)
+			_, err = volumedriver.Enumerate(&api.VolumeLocator{}, nil)
 			Expect(err).ToNot(HaveOccurred())
-			numVolumesAfter = len(volumes)
```

Comment:
correct fix should be to use `numVolumesAfter` and testing for it instead of ignoring them.

The previous test was not doing so.

Reply:
ok